modules/exploits/multi/browser/chrome_jscreate_sideeffect.rb

```diff
@@ -0,0 +1,370 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Chrome 80 JSCreate side-effect type confusion exploit',
+      'Description'    => %q{
+      This module exploits an issue in Chrome 80.0.3987.87 (64 bit) on Windows 10 --no-sandbox.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+          'saelo', # discovery
+          'István Kurucsai', # exploit
+          'timwr', # metasploit copynpasta
+        ],
+      'References'     => [
+          ['CVE', '2020-6418'],
+          ['URL', 'https://bugs.chromium.org/p/chromium/issues/detail?id=1053604'],
+          ['URL', 'https://blog.exodusintel.com/2020/02/24/a-eulogy-for-patch-gapping'],
+          ['URL', 'https://ray-cp.github.io/archivers/browser-pwn-cve-2020-6418%E6%BC%8F%E6%B4%9E%E5%88%86%E6%9E%90'],
+        ],
+      'Arch'           => [ ARCH_X64 ],
+      'Platform'       => ['windows','osx'],
+      'DefaultTarget'  => 0,
+      'Targets'        => [ [ 'Automatic', { } ] ],
```

Comment:
```suggestion
      'Targets'        => 
        [
          ['Windows 10 Google Chrome 80.0.3987.87 (64 bit)', {}]
        ],
```
Would this still work or am I being pedantic?